templates/prescription/general_edit.html

```diff
@@ -4,21 +4,42 @@
 
 <link rel="stylesheet" href="{$CSS_HEADER}" type="text/css">
 <link rel="stylesheet" href="{$WEBROOT}/interface/themes/jquery.autocomplete.css" type="text/css">
+<link rel="stylesheet" href="{$WEBROOT}/library/js/jquery-ui.min.css" type="text/css" /> 
+
+
 {literal}
 <style type="text/css">
-    .text {
-        font-size: 9pt;
-    }
+
+
+input{
+   margin: 5px;
+}
+select{
+  margin: 5px;
+}
 </style>
 {/literal}
 <script language="Javascript">
+
 {literal}
-		function my_process () {
-			// Pass the variable
-			opener.document.prescribe.drug.value = document.lookup.drug.value;
-			// Close the window
-			window.self.close();
-		}
+    function my_process () {
+      // Pass the variable
+      opener.document.prescribe.drug.value = document.lookup.drug.value;
+      // Close the window
+      window.self.close();
+    }
+    function validateForm() {
+    var x = document.forms["prescribe"]["dosage"].value;
+  var y = document.forms["prescribe"]["quantity"].value;
+    if (x == null || x == "") {
+        alert("<?php print xls('Dosage must be filled out')?>");
+        return false;
+    }
+  if (y == null || y == "") {
+        alert("<?php print xls('Quantity must be filled out')?>");
+        return false;
+    }
+}
```

Comment:
Is this new validateForm even needed? Considering the uncommenting of the CheckRequired stuff below?